core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceRepository.scala

```diff
@@ -41,7 +72,43 @@ class WorkspaceRepository(dataSource: SlickDataSource) {
       access.workspaceQuery.delete(workspace.toWorkspaceName)
     }
 
+  def deleteWorkspaceRecord(workspaceName: WorkspaceName): Future[Boolean] =
```

Comment:
Nit: Call this `deleteWorkspace` to remain consistent with the naming scheme in the rest of this class (and to retain the repository abstraction over the data layer that we're dealing with objects and not records).